junit-platform-gradle-plugin/src/test/groovy/org/junit/platform/gradle/plugin/JUnitPlatformPluginSpec.groovy

```diff
@@ -83,6 +86,59 @@ class JUnitPlatformPluginSpec extends Specification {
 		noExceptionThrown()
 	}
 
+	@CompileStatic
+	def "setting junitPlatform properties statically"() {
+		/*
+		 * This test ensures that the extensions can be accessed as expected in statically typed languages like Kotlin.
+		 * The `Action` syntax is much nice in kotlin and doesn't require `it` everywhere.
+		 * https://github.com/junit-team/junit5/issues/902
+		 */
+		given:
+		// Goofy syntax required because of closure overload
+		project.apply ({ObjectConfigurationAction it ->
+			it.plugin('org.junit.platform.gradle.plugin')
+		})
+		when:
+		JUnitPlatformExtension junitPlatform = project.extensions.getByType(JUnitPlatformExtension)
+		junitPlatform.platformVersion = '5.0.0-M1'
+		junitPlatform.enableStandardTestTask = true
+		junitPlatform.logManager = 'org.apache.logging.log4j.jul.LogManager'
+		junitPlatform.selectors {
+			it.uris'u:foo', 'u:bar'
```

Comment:
Aren't they aren't currently unnecessary in the kotlin DSL? Do you mean the next version of the groovy DSL?